django/db/models/functions/window.py

```diff
@@ -0,0 +1,125 @@
+from django.db.models import FloatField, Func, IntegerField
+
+__all__ = [
+    'CumeDist', 'DenseRank', 'FirstValue', 'Lag', 'LastValue', 'Lead',
+    'NthValue', 'Ntile', 'PercentRank', 'Rank', 'RowNumber',
+]
+
+
+class CumeDist(Func):
+    function = 'CUME_DIST'
+    name = 'CumeDist'
+    window_compatible = True
+
+    def __init__(self, *expressions, **extra):
+        super().__init__(*expressions, output_field=FloatField(), **extra)
```

Comment:
You can update this after 08654a99bbdd09049d682ae57cc94241534b29f0, I believe.

Reply:
Yes, I just noticed after I did a `pull` from upstream.